frame/staking/src/lib.rs

```diff
@@ -697,9 +813,31 @@ decl_storage! {
 		pub Stakers get(fn stakers):
 			map hasher(blake2_256) T::AccountId => Exposure<T::AccountId, BalanceOf<T>>;
 
+		/// Snapshot of validators at the beginning of the current election window. This should only
```

Comment:
Surprisingly, it is actually _not_ better to store these in a struct. They are decoded individually.